src/bodse/timetable/GTFS_download.py

```diff
@@ -0,0 +1,107 @@
+import requests
+from datetime import datetime, timedelta
+from pathlib import Path
+import zipfile
+import os
+from clint.textui import progress
+import sys
+
+
+def download_gtfs_feed(directory):
+    current_datetime = datetime.now()
+    current_date_time = current_datetime.strftime("%Y%m%d_%H-%M-%S")
+
+    try:
+        print("Downloading started")
+        url = "https://data.bus-data.dft.gov.uk/timetable/download/gtfs-file/all"
+
+        req = requests.get(url, stream=True)
+
+        filename = Path("GTFS_" + current_date_time + ".zip")
+        filepath = directory / filename
```

Comment:
This can be simplified using f-strings to
```suggestion
        filepath = directory / f"GTFS_{current_date_time}.zip"
```